app/src/main/res/values/dimens.xml

```diff
@@ -22,7 +22,7 @@
     <dimen name="text_16sp">16sp</dimen>
     <dimen name="text_20sp">20sp</dimen>
     <dimen name="text_24sp">24sp</dimen>
-
+    <dimen name="text_drawable_12dp">12dp</dimen>
```

Comment:
I think `icon_12dp` is better because it's easier to search when I specify icon size.
I'll adjust them later!